test_conformance/spirv_new/spirv_asm/spv1.4/image_operand_signextend.spvasm32

```diff
@@ -0,0 +1,38 @@
+; SPIR-V
+; Version: 1.4
+; Generator: Khronos LLVM/SPIR-V Translator; 14
+; Bound: 31
+; Schema: 0
+               OpCapability Addresses
+               OpCapability Linkage
+               OpCapability Kernel
+               OpCapability ImageBasic
+               OpCapability LiteralSampler
+          %1 = OpExtInstImport "OpenCL.std"
```

Comment:
Do we need this?

Reply:
Nope!  Removed from all of the spvasm files.